addons/survey/models/survey_survey.py

```diff
@@ -105,10 +106,12 @@ def _compute_users_can_signup(self):
     @api.depends('user_input_ids.state', 'user_input_ids.test_entry')
     def _compute_survey_statistic(self):
         stat = dict((cid, dict(invite_count=0, answer_count=0, answer_done_count=0)) for cid in self.ids)
-        UserInput = self.env['survey.user_input']
-        base_domain = ['&', ('survey_id', 'in', self.ids), ('test_entry', '!=', True)]
-
-        read_group_res = UserInput.read_group(base_domain, ['input_type', 'state'], ['survey_id', 'input_type', 'state'], lazy=False)
+        read_group_res = self.env['survey.user_input'].read_group(
```

Comment:
Unnecessary diff

Reply:
I did some cleaning but since this targets stable it should contain minimal diff indeed, I reverted it.